packages/web-frontend/public/index.html

```diff
@@ -24,6 +24,8 @@
     />
     <meta property="og:url" content="https://tupaia.org" />
     <meta property="og:title" content="Tupaia" />
+
+    <% if (process.env.NODE_ENV === 'production') { %>
```

Comment:
The [Dan Abramov](https://github.com/facebook/create-react-app/issues/3112) approved method.

Reply:
Heh, love that guy. This actually won't help for dev and feature instances (which are also built in prod mode), but no harm in having it there to avoid it running locally.